src/renderable/animationsheet.js

```diff
@@ -143,7 +143,7 @@
                             "string parameters for addAnimation are not allowed for standard spritesheet based Texture"
                         );
                     } else {
-                        this.anim[name].frame[i] = this.textureAtlas[this.atlasIndices[index[i]]];
+                        this.anim[name].frame[i] = Object.assign(this.textureAtlas[this.atlasIndices[index[i]]], { delay: this.anim[name].animationspeed });
```

Comment:
I think this should be:

```javascript
this.anim[name].frame[i] = Object.assign(
    {},
    { "delay" : this.anim[name].animationspeed },
    this.textureAtlas[this.atlasIndices[index[i]]]
);
```

Based on the properties of `Object.assign`;

* Don't update the underlying atlas structure (first arg is empty Object)
* Don't override `delay` property if it exists on the atlas (`{ "delay" : ... }` Object before frame Object)

And also broken into multiple lines, because it's getting kind of long.

(Should have the same change on the diff above.)